.github/workflows/github.yml

```diff
@@ -1,14 +1,18 @@
+# This GitHub Actions workflow ensures that any pull request targeting the main or main-*
+# branches meets certain criteria before it is merged. The workflow sets up the necessary infrastructure,
+# configures the environment, and performs checks on the pull request title, body, labels
+# and assignee to ensure that it adheres to predefined standards. If any of these checks fail,
+# the workflow will exit with an error, preventing the pull request from proceeding.
+
 name: GitHub
-run-name: 🔬Pull request inspection on ${{ github.event.number }}
+run-name: 🔬 Pull request inspection on ${{ github.event.number }}
 
 on:
   pull_request:
-    branches:
-      - main
-      - "release-*"
+    branches: [main, main-*]
```

Comment:
Why the change from `release-*` to `main-*`?

Reply:
We only use `main-*` branches in `APIM` not `release-*`